src/commands/profiles/create.js

```diff
@@ -170,7 +170,8 @@ class ProfilesCreate extends BaseCommand {
   }
 
   async saveCredentials() {
-    const apiKeyFriendlyName = `twilio-cli for ${os.userInfo().username} on ${os.hostname()}`;
+    let apiKeyFriendlyName = `twilio-cli for ${os.userInfo().username} on ${os.hostname()}`;
+    apiKeyFriendlyName = apiKeyFriendlyName.substring(0, 64);
```

Comment:
If we will not allow `apiKeyFriendlyName` to be changed, we might want to keep it as a `const`.

Reply:
I wanted to break up lines 173 and 174, which means `apiKeyFriendlyName` is modified on 174.